lore/settings.py

```diff
@@ -348,3 +348,4 @@ def get_var(name, default):
     }
 }
 HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
+XANALYTICS_URL = get_var('XANALYTICS_URL', "")
```

Comment:
I don't know yet. We haven't received any details of the other end. I assume there will be tweaking necessary in any case once the API is released.